backend/app/utils/api_response.py

```diff
@@ -0,0 +1,47 @@
+"""API response classes."""
+from abc import ABC, abstractmethod
+from typing import Any, Dict
+
+class ApiResponse(ABC):
+    """Abstract base class for API responses."""
+
+    @abstractmethod
+    def to_dict(self) -> Dict[str, Any]:
+        """Convert the response to a dictionary."""
+        pass
+
+class SuccessResponse(ApiResponse):
+    """API response for successful operations."""
+
+    def __init__(self, data: Any = None, message: str = "Success"):
+        """Initialize the success response."""
+        self.status = "success"
```

Comment:
add more attributes like response_generated_at, pagination attribute(only applicable for list responses) and better to rename data to payload(as data is generic)

- [ ] Add more attributes 
- [ ] Rename data to payload